spotify_dl/spotify_dl.py

```diff
@@ -45,6 +45,8 @@ def spotify_dl():
                         help="Spotify Playlist link URL")
     parser.add_argument('-s', '--scrape', action="store",
                         help="Use HTML Scraper for YouTube Search", default=True)
+    parser.add_argument('-I', '--invidious', action="store_true",

```

Comment:
lowercase I please

Reply:
lowercase i is already used for uri parameter